cli/src/commonMain/kotlin/utils/PrintHelper.kt

```diff
@@ -41,5 +43,19 @@ internal fun SuspendingCliktCommand.echoError(message: String?) {
     }
 }
 
+/**
+ * Print the [message] to stdout. If required as a json object.
+ */
+internal fun SuspendingCliktCommand.echoMessage(message: CliPrintable) {
+    if (useJsonFormat) {
+        echo(message.json())
+    } else {
+        // TODO: Replace with human readable formatting.
+        echo(message.toString())
```

Comment:
The `MessagePrintable` does not override `toString()`, does this actually work?

Reply:
Yes, it prints the object with its values.
Nothing I want to keep, but as the message is changed in the following commit, I think that it is good enough.